builder/index_full.html

```diff
@@ -49,7 +49,7 @@
                             <div class="col-auto rounded order-xl-0 order-0">
                                 <div class="row h-100 dark-shadow dark-6 rounded" id="helmet-dropdown">
                                     <div class="col-auto px-lg-1 g-0 dark-7 rounded-end my-auto text-center scaled-item-icon" id="helmet-img-loc">
-                                        <img id="helmet-img" class="img-fluid rounded" src="../media/items/new/generic-helmet.png">
+                                        <div id="helmet-img" class="img-fluid rounded" style="background-image: url('../media/items/new.png'); image-rendering: pixelated; background-size: 1200% 100%; background-position: 45.45454545454546% 0; aspect-ratio: 1/1;"></div>
```

Comment:
might help reduce code duplication if a css class is made for the "equipment atlas" images, and then inline style is used only for positioning. Though that might be more confusing for future code readers